social_platform/serializers.py

```diff
@@ -27,4 +27,22 @@ class AccountUpdateSerializer(serializers.Serializer):
     new_roles=serializers.IntegerField(required=False)
 class AccountDeleteSerializer(serializers.Serializer):
```

Comment:
why are you used, same things in three serializer?
makes a common serializer and used it where required.